common/helpers.js

```diff
@@ -6,7 +6,7 @@ const moment = require('moment');
 const dataGuidsHostPrefix = 'dg.';
 const dosDataObjectPathPrefix = '/ga4gh/dos/v1/dataobjects/';
 const drsDataObjectPathPrefix = '/ga4gh/drs/v1/objects/';
-const jadeDataRepoHostRegex = /jade.*\.datarepo-.*\.broadinstitute\.org/;
+const jadeDataRepoHostRegex = /.*data.*[-.](broadinstitute\.org|terra\.bio)$/;
```

Comment:
Should Rawls also update now that we've committed the sin of copy-pasting this code?

https://github.com/broadinstitute/rawls/blob/0b60a4b27772c012fe098e48d437cac863820c10/core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/martha/MarthaModel.scala#L21

Reply:
Looks like it, yes.